lib/game.rb

```diff
@@ -0,0 +1,43 @@
+class Game
+  def initialize
+    @rolls = [] 
+  end
+
+  def roll(pins)
```

Comment:
I've noticed this code only works for full games, which is fine but it does mean it's quite hard to test in irb, you have to enter every roll one-after-another. I'd suggest making your life easier by having some sort of function that can be called on to input a list of rolls immediately, something like

def test
[1, 4, 4, 5, 6, 4, 5, 5, 10, 0, 1, 7, 3, 6, 4, 10, 2, 8, 6].each { |my_roll| mygame.roll(my_roll) }
end